simulator/src/test/java/com/hazelcast/simulator/probes/impl/HdrProbeTest.java

```diff
@@ -26,16 +30,19 @@ public void testConstructor_noThroughputProbe() {
         assertFalse(tmpProbe.isPartOfTotalThroughput());
     }
 
+    // this test is fragile and can easily fail if there is some kind of pausing going on
```

Comment:
It also seems to fail on CloudBees: https://hazelcast-l337.ci.cloudbees.com/job/Simulator-pr-builder/com.hazelcast.simulator$simulator/127/testReport/junit/com.hazelcast.simulator.probes.impl/HdrProbeTest/testDone_withExternalStarted/